pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -539,17 +459,13 @@ in with lib.licenses;
     depsBuildBuild = [ buildPackages.stdenv.cc ];
   };
 
-  mame = mkLibRetroCore {
+  mame = mkLibRetroCore rec {
     core = "mame";
-    src = fetchRetro {
-      repo = "mame";
-      rev = "ed987ad07964a938351ff3cc1ad42e02ffd2af6d";
-      sha256 = "0qc66mvraffx6ws972skx3wgblich17q6z42798qn13q1a264p4j";
-    };
+    src = getCoreSrc core;
     description = "Port of MAME to libretro";
     license = gpl2Plus;
 
-    extraBuildInputs = [ alsa-lib libGLU libGL portaudio python27 xorg.libX11 ];
+    extraBuildInputs = [ alsa-lib libGLU libGL portaudio python3 xorg.libX11 ];
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** :medal_military:  :medal_military:  :medal_military: